.gitignore

```diff
@@ -5,3 +5,4 @@ examples/*.peggy
 examples/*.pest
 node_modules
 t.abnf
+.cert/
```

Comment:
unrelated?

Reply:
yes, its unrelated, but intentional.  I've started using [hostlocal](https://github.com/hildjj/hostlocal#readme) in local workflows.